File: android/app/src/debug/java/com/laneshadow/sandbox/stories/organisms/OrganismStories.kt
package com.laneshadow.sandbox.stories.organisms

import com.nativesandbox.model.Story

object OrganismStories {
    val all: List<Story> = buildList {
        addAll(LSTopBarStory.all)
        addAll(LSNavBarStory.all)
        addAll(LSSectionHeaderStory.all)
        addAll(LSMapLayerStory.all)
        addAll(LSRouteSheetStory.all)
        addAll(LSSessionsDrawerStory.all)
        addAll(LSRouteCardStory.all)
        addAll(LSNavigatorMessageStory.all)
        addAll(LSInlineErrorCalloutStory.all)
        addAll(LSMapControlsStory.all)
    }
}

File: ios/LaneShadow/Sandbox/Stories/Organisms/OrganismStories.swift
import LaneShadowTheme
import NativeSandbox
import SwiftUI

/// Organism stories — Sprint 05 organism components.
@MainActor
enum OrganismStories {
    static let all: [Story] = {
        var stories: [Story] = []
        stories += LSTopBarStory.all
        stories += LSNavBarStory.all
        stories += LSSectionHeaderStory.all
        stories += LSNavigatorMessageStory.all
        stories += LSInlineErrorCalloutStory.all
        stories += LSMapLayerStory.all
        stories += LSRouteSheetStory.all
        stories += LSSessionsDrawerStory.all
        return stories
    }()
}
